kong/plugins/aws-lambda/schema.lua

```diff
@@ -14,5 +14,6 @@ return {
     log_type = {type = "string", required = true, default = "Tail", 
                        enum = {"Tail", "None"}},
     port = { type = "number", default = 443 },
+    unhandled_status = {type = "number"},
```

Comment:
👍 Awesome, sad I removed the validation in one of my earlier commits now.

I'm happy to add spaces. Just getting the hang of this lua thing. 😉